client/src/pages/tasks/Form.js

```diff
@@ -362,22 +362,11 @@ const TaskForm = ({ edit, title, initialValues }) => {
                         name="assessment_customFieldEnum1"
                         label={`Assessment of ${Settings.fields.task.customFieldEnum1.label}`}
                         component={FieldHelper.SpecialField}
-                        onChange={value =>
+                        value={values.assessment_customFieldEnum1}
```

Comment:
Setting the value is not wanted; the users are expected to fill in a fresh assessment when they change `customFieldEnum1` (**Project status** in development); this text (and the change in status) are recorded in a `CHANGE_RECORD` note.